### PR TITLE
Add mask-icon for pinned safari tab

### PR DESCRIPTION
**Pull request by MaxLeiter**
https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html

Look in top left of image:
image

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 	<link rel="shortcut icon" href="img/favicon.png" data-other="img/favicon-notification.png" data-toggled="false" id="favicon">
 	<link rel="apple-touch-icon" sizes="120x120" href="img/apple-touch-icon-120x120.png">
+	<link rel="mask-icon" href="img/logo.svg" color="black">
```
Comment (astorije) [inline client/index.html]: Can we use same rgb hex than the theme-color one instead?

**Comment by MaxLeiter** [comment]
@astorije not a super big difference, but if you prefer it let me know and i'll update PR:

image

**Comment by astorije** [comment]
That'd be great, thanks! If anything, for consistency and potential future
copy-paste :)

**Comment by MaxLeiter** [comment]
Done

**Pull request merged**